### PR TITLE
fixes the semitransparent stroke disabled

### LINKED ISSUE #5951: Semitransparent strokes disabled in WebGL mode
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [X] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [X] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### p5.js version

1.5.0

### Web browser and version

Firefox 108.0

### Operating System

MacOS 12.5.1

### Steps to reproduce this

Noticed this one after someone on the p5 discord was asking about it.

In this bit of code, we make WebGL strokes fully opaque: https://github.com/processing/p5.js/blob/00821f33ca1d8a6990364568f0374c4aaf713faa/src/webgl/p5.RendererGL.js#L670-L672

Are we able to remove this without causing any bugs? To test it, one can wrap color arguments in `color`, e.g. `stroke(color(0, 255, 0, 50))`. If not, maybe we should document this, as it currently can leave people wondering what they're doing wrong when switching from 2D to WebGL mode.

### PR DESCRIPTION
**Pull request by Prateek462003**
Resolves #5951

 Changes:
  fixed the stroke method . previously it was using the default opaque strokes even if changed by the user

#### PR Checklist

- [ x] `npm run lint` passes
- [ ] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by inaridarkfox4231** [comment]
I don't understand why the warning is needed when it should just allow transparency changes.

**Comment by Prateek462003** [comment]
@inaridarkfox4231 I thought it would be better if the user would know that little detail . I will push the changes right away.

**Review by davepagurek — COMMENTED** [review]
Just some small cleanup, otherwise this is good to go!

We might want to consider making a follow-up issue to address the behaviour strokes overlapping with each other at corners though. Here's a rectangle:
image

...but even without that solved, this should still enable people to draw single lines with transparency!

File: src/webgl/p5.RendererGL.js [hunk 1]
```diff
@@ -703,9 +703,12 @@ p5.RendererGL.prototype.fill = function(v1, v2, v3, a) {
  * black canvas with purple cube with pink outline spinning
  */
 p5.RendererGL.prototype.stroke = function(r, g, b, a) {
-  //@todo allow transparency in stroking currently doesn't have
-  //any impact and causes problems with specularMaterial
-  arguments[3] = 255;
+  // @todo allow transparency in stroking currently doesn't have
+  // any impact and causes problems with specularMaterial
+  if(arguments[3]===undefined){
```
[the remark this answers is not in the document]
Comment (davepagurek) [reply src/webgl/p5.RendererGL.js:708]: Oh I think we can also take out that `@todo` comment too. This doesn't seem to affect specularMaterial any more (aside from the stroke joint blending issues I mentioned earlier, which we can address separately.)

**Comment by Prateek462003** [comment]
@davepagurek please review this

**Pull request closed without merging**